CHANGELOG.md

```diff
@@ -38,20 +38,45 @@ This text is announced by screen readers when the character count input is focus
 
 This was added in [pull request #2742: Add ability to customise character count fallback text](https://github.com/alphagov/govuk-frontend/pull/2742).
 
+#### Localise the character count's counter message
+
+You can now translate the text shown by the [character count](https://design-system.service.gov.uk/components/character-count/) component to inform users of:
+
+- when they reached the maximum number of characters or words
+- the number of characters or words over or under that maximum
+
+The Nunjucks macro accepts new options to let you customise each message:
```

Comment:
```suggestion
The Nunjucks macro accepts customisation for each message:
```

Reply:
My intent was to highlight that people would be able to pass new options to their Nunjucks macro, and this was a bit lost there. Would something like the following work?

```suggestion
The Nunjucks macro accepts new options to customise each message:
```

Reply:
Ah I see. I've edited my suggestion so the only change is from passive to active voice. Does that work?